src/library/scala/Option.scala

```diff
@@ -116,14 +116,38 @@ sealed abstract class Option[+A] extends Product with Serializable {
   self =>
 
   /** Returns true if the option is $none, false otherwise.
+   *
+   * This is equivalent to:
```

Comment:
The edits don't actually respect the surrounding text alignment. https://docs.scala-lang.org/style/scaladoc.html

Reply:
I'm not sure I follow.  You're talking about what column the "gutter asterisks" are?  I believe I followed the convention throughout the file.